### PR TITLE
Varvara/feature/sass

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

**Review by kipod — APPROVED** [review]
good

File: src/components/Queue/Queue.tsx [hunk 1]
```diff
@@ -119,18 +122,21 @@ export default function Queue(): ReactElement {
             </div>
           ))
           :
-          <div className="infoMessageIntake">NO PATIENTS IN QUEUE</div>
+          <div className="infoMessage">NO PATIENTS IN QUEUE</div>
         }
 
         <button className="queue_add_button" onClick={() => {
+          console.log("Add patient");
```
Comment (bezryma) [inline src/components/Queue/Queue.tsx:145]: we do not like left log in code :)

File: src/App.tsx [hunk 2]
```diff
@@ -48,9 +48,7 @@ function App() {
               )}
           </Route>
           <Route path="/password-choose/:api_key" children={<PasswordChoose />} />
-          {/* <Route path="/password-choose/:api_key">
-            <PasswordChoose />
-          </Route> */}
+
```
Comment (bezryma) [inline src/App.tsx]: if it is an empty line, lets remove it!

**Review by bezryma — COMMENTED** [review]
Looks great!

File: src/components/Queue/Queue.tsx (hunk 1 above)
Comment (varvarashcherbyna7) [reply src/components/Queue/Queue.tsx:145]: Thanks )
I think to apply later a special package which will remove all logs

File: src/App.tsx (hunk 2 above)
Comment (varvarashcherbyna7) [reply src/App.tsx]: Thank you for reminding me,  I have a problem with this !)

**Pull request merged**